platform/atarixl/Makefile.atarixl

```diff
@@ -35,7 +35,7 @@ CONTIKI_CPU = $(CONTIKI)/cpu/6502
 include $(CONTIKI_CPU)/Makefile.6502
 
 SHADOW_RAM_SOURCEFILES  = ethernet.c ethernet-drv.c timer.c uip.c uiplib.c
-SHADOW_RAM2_SOURCEFILES = clock.c uip_arch.c uip_arp.c
+SHADOW_RAM2_SOURCEFILES = clock.c uip_arch.c uip_arp.c unload.c random.c
```

Comment:
Please look at https://github.com/contiki-os/contiki/pull/1265/files#diff-a59eec8547717a09a0ad24ff500a7600

I needed to move some objects, in order to make PR #1265 fit into the apple2enh and atarixl platforms.  I hope that you will include those moves here (or, make an even better fitting map for that PR).